src/config/2020.json

```diff
@@ -0,0 +1,225 @@
+{
+  "settings": [
+    {
+      "is_live": false,
+      "supported_languages": ["en","es","fr","ja"],
+      "ebook_languages": []
+    }
+  ],
+  "outline": [
```

Comment:
Let's omit this for now until we have an actual 2020 outline. I have a question later about navigating to years' home pages consistently which might make this unnecessary for now.

Reply:
I still think it's useful for have the ability to distinguish between a chapter that doesn't exist yet but should (302) and one that shouldn't and has been typed wrong (404). Even if we do change the navigation. Of course we can keep that logic and just not have chapters configured yet, but makes testing that painful (reminder to self - we should write test cases fo this!).

Also reminds us of the structure to save us having to copy it later.

So it is useful, and I don't see any reason or harm to have this in here now. So my preference would be to leave it.